src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -26,26 +29,60 @@ public class PostHearingReviewAboutToSubmitHandler implements PreSubmitCallbackH
-
         return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
             && callback.getEvent() == EventType.POST_HEARING_REVIEW
             && isPostHearingsEnabled;
     }
 
     @Override
     public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
-                                                          String userAuthorisation) {
+        String userAuthorisation) {
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
 
+        updateCaseStatus(caseData);
         PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(caseData);
 
-        String caseId = caseData.getCcdCaseId();
+        excludePanelMembers(caseData, response);
 
+        String caseId = caseData.getCcdCaseId();
         PostHearingReviewType typeSelected = caseData.getPostHearing().getReviewType();
         log.info("Review Post Hearing App: handling action {} for case {}", typeSelected,  caseId);
 
         SscsUtil.clearDocumentTransientFields(caseData);
 
         return response;
     }
+
+    protected void updateCaseStatus(SscsCaseData caseData) {
+        PostHearing postHearing = caseData.getPostHearing();
+        if (nonNull(postHearing) && nonNull(postHearing.getSetAside().getAction())) {
+            caseData.setState(State.NOT_LISTABLE);
+            caseData.setInterlocReviewState(InterlocReviewState.AWAITING_ADMIN_ACTION);
+        }
+    }
+
+    protected void excludePanelMembers(SscsCaseData caseData, PreSubmitCallbackResponse<SscsCaseData> response) {
+        PostHearing postHearing = response.getData().getPostHearing();
+        if (PostHearingReviewType.SET_ASIDE.equals(postHearing.getReviewType())
+            && SetAsideActions.GRANT.equals(postHearing.getSetAside().getAction())
+        ) {
+            log.info("Set Aside is granted. Excluding panel members for case {}", caseData.getCcdCaseId());
+
+            HearingDetails latestHearing = caseData.getLatestHearing().getValue();
+            List<String> panelMemberIds = latestHearing.getPanelMemberIds(); // TODO test this list to confirm working as expected
+            String judgeId = latestHearing.getJudgeId();
+            ArrayList<JudicialUserBase> panelMembersToExclude = panelMemberIds.stream()
+                .map(id -> JudicialUserBase.builder().personalCode(id).build())
+                .collect(Collectors.toCollection(ArrayList::new));
+            JudicialUserBase judgeToExclude = JudicialUserBase.builder().personalCode(judgeId).build();
+            panelMembersToExclude.add(judgeToExclude);
+
+            PanelMemberExclusions exclusions = caseData.getSchedulingAndListingFields().getPanelMemberExclusions();
+            SscsUtil.excludePanelMembers(exclusions, panelMembersToExclude);
+
+            caseData.setPanel(null);
+            exclusions.setArePanelMembersReserved(YesNo.NO);
+            exclusions.setArePanelMembersExcluded(YesNo.YES);
```

Comment:
Remove this line. SscsUtil.excludePanelMembers will set this to yes

Reply:
removed